core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/StreamletPort.scala

```diff
@@ -50,6 +50,22 @@ trait Inlet extends StreamletPort
  */
 trait Outlet extends StreamletPort
 
+/**
+ * A default error handler. This error handler just logs bad message and skips them.
```

Comment:
comment should go to logAndSkip method

Reply:
Agree. Currently, I'm thinking that we should log one error. Not two